.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!
+
+This PR template was copied from https://github.com/testing-library/react-testing-library/blob/main/.github/PULL_REQUEST_TEMPLATE.md
+
+Please fill out the information below to expedite the review and (hopefully)
+merge of your pull request!
+-->
+
+<!-- What changes are being made? (What feature/bug is being fixed here?) -->
+
+**What**:
+
+<!-- Why are these changes necessary? -->
+
+**Why**:
+
+<!-- How were these changes implemented? -->
+
+**How**:
+
+<!-- Have you done all of these things?  -->
+
+**How to test it**:
+
+<!-- If applicable, add the steps to test your changes -->
+
+**Checklist**:
+
+<!-- add "N/A" to the end of each line that's irrelevant to your changes -->
+
+<!-- to check an item, place an "x" in the box like so: "- [x] Documentation" -->
+
+- [ ] Documentation added
```

Comment:
What is meant by documentation?

Reply:
This is copied from a library, so it's more suitable for that use case.

Anyway, I think we should be a little more documentation-oriented like we did with the network support (https://github.com/gnosis/safe-react/blob/development/docs/networks.md)

Reply:
This item will be updated once each a few months. I would remove it as it generally won't be used.

Reply:
Ok, let's make it a policy if that would be the case.